### DISCUSSION TITLE
mumu的退出有问题

### DISCUSSION THREAD

**Discussion by 1458963** [MaaAssistantArknights/MaaAssistantArknights#7644] [Q&A] [answered]
有亿点糊，应该能看出来
https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/153965315/a969dcf1-166a-4a5a-bc7c-440673261201

**Comment by ABA2396** [comment]
去issue里发下日志

**Reply by 1458963** [reply]
哪里

**Reply by ABA2396** [reply, marked as answer]
设置-关于我们-问题反馈
